com.xargon.retrodeck.yml

```diff
@@ -0,0 +1,717 @@
+app-id: com.xargon.retrodeck
```

Comment:
the domain `xargon.com` seems to just be domain squatted. I don't think this fit the criteria for app-id.

Reply:
It`s not realted with that website, atm my only website is github, a new one is in the making

Reply:
then why are you using it?